runtime/kusama/src/governance/tracks.rs

```diff
@@ -268,10 +268,10 @@ const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 15
 			name: "big_spender",
 			max_deciding: 10,
 			decision_deposit: 5 * GRAND,
-			prepare_period: 4,
+			prepare_period: 4 * HOURS,
 			decision_period: 28 * DAYS,
```

Comment:
any chance we can set this to 7 DAYS for all spenders and tippers? The curves are quite demanding on the support threshold. 28 days means a lot of planning and exchange rate uncertainty, which is harmful for ecosystem dev teams

Reply:
Reduced to 14 days throughout, with tippers at 7 days.

The curves are designed to be safe. If a faster conclusion is desired by teams, then I think they really ought to be helping increase turnout for their proposal from token holders/voters.